library/Fission/Internal/Orphanage/ByteString.hs

```diff
@@ -0,0 +1,11 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
+
+module Fission.Internal.Orphanage.ByteString () where
+
+import Fission.Prelude
+
+instance FromJSON ByteString where
+  parseJSON = withText "ByteString" (pure . encodeUtf8)
+
+instance ToJSON ByteString where
+  toJSON = String . decodeUtf8Lenient
```

Comment:
Where do we want these? There's a good chance that we actually want a `newtype` wrapper for the thing.

Reply:
They're used in `Fission.Web.Auth.JWT.Types` for serializing/deserializing JWTs

Reply:
yeah I can see that we might want `newtype`s for some of these. especially a `DID` type